src/spreadsheet-jsonify.js

```diff
@@ -0,0 +1,50 @@
+/**
+This program fetches the club data from a public spreadsheet and converts it into
+an array of the following form:
+
+[
+  {name: "Counterfeit Money Club", description: "Officer: You're under arrest for using counterfeit bank notes. Me: *Slides a crisp £17 bill* How about now eh?", categories:
```

Comment:
lol

Reply:
I should clarify that categories is a STRING, not an array. Didn't realize that was like the most important part of this code... -_-